Docs/user_guide/getting_started.md

```diff
@@ -149,3 +149,51 @@ Browse the [User Guide](user_interface) section to find quick overview of the ap
+- Geometry: Specify location and shape of an object in 3D space.
+- Intensity: Typically refers to value of a voxel.
+- IJK: Voxel coordinate system axes. Each integer coordinate value combination corresponds to a voxel center.
+- ITK: [Insight Toolkit](https://itk.org/). Software library that Slicer uses for most image processing operations.
+- Labelmap (or labelmap volume, labelmap volume node): Volume node that has discrete (integer) voxel values. Typically each value corresponds to a specific structure or region. This allows compact representation of non-overlapping regions in a single 3D array. Most software use a single labelmap to store an image segmentation, but Slicer uses a dedicated segmentation node, which can contain multiple representations (multiple labelmaps to allow storing overlapping segments; closed surface representation for quick 3D visualization, etc.).
+- LPS: Left-posterior-superior anatomical coordinate system. Most commonly used coordinate system in medical image computing. Slicer stores all data in LPS coordinate system on disk (and converts to/from RAS when writing to or reading from disk).
+- Markups: Simple geometric objects and measurements that the user can place in viewers. [Markups module](modules/markups) can be used to create such objects. There are several types, such as fiducial list, line, curve, plane, ROI.
+- Master volume: Voxel values of this volume is used during segmentation by those effects that rely on intensity of an underlying volume.
+- [MRML (Medical Reality Markup Language)](https://en.wikipedia.org/wiki/Medical_Reality_Markup_Language): Software library for storage, visualization, and processing of information objects that may be used in medical applications. The library is designed to be reusable in various software applications, but 3D Slicer is the only major application that is known to use it.
+- Model (or model node): MRML node storing surface mesh (consists of triangle, polygon, or other 2D cells) or volumetric mesh (consists of tetrehedral, wedge, or other 3D cells)
+- Module: A Slicer module is a software component consisting of a graphical user interface (that is displayed in the module panel when the module is selected), a logic (that implements algorithms that operate on MRML nodes), and may provide new MRML node types, displayable managers (that are responsible for displaying those nodes in views), input/output plugins (that are responsible for load/save MRML nodes in files), and various other plugins. Modules are typically independent and only communicate with each other via modifying MRML nodes, but sometimes a module use features provided by other modules by calling methods in its logic.
+- Node (or MRML node): One data object in the scene. A node can represent data (such as an image or a mesh), describe how it is displayed (color, opacity, etc.), stored on disk, spatial transformations applied on them, etc.
```

Comment:
Add:
There is a C++ class hierarchy to define the common behaviors of nodes, such as the property of being storable on disk or being geometrically transformable.   The structure of this class hierarchy can be inspected in the code or [in the API documentation](http://apidocs.slicer.org/master/classvtkMRMLStorableNode.html).